woocommerce/woocommerce-ios
# Storage: Product details

This PR adds product details  support to **model v12** which is scheduled to ship with WCiOS v1.5:

<img width="498" alt="Screen Shot 2019-03-20 at 1 55 36 PM" src="https://user-images.githubusercontent.com/154014/54716565-023d7a00-4b24-11e9-8d7c-b89604cfe11b.png">

Nothing really earth shattering here, just updates the the model and all of the generated `NSManagedObject` subclasses for `Product`.

Ref: #764

## Testing

1. Review the model + code
2. Run the unit tests — verify they are ✅ 
3. Build and run the app — verify nothing strange is going on
4. Verify the lightweight migration works from v11 to v12 (it should since we are simply adding new entities in this PR).

@ctarda would you mind giving this a quick review?

/cc @astralbodies because core data 🥇
